xds/internal/httpfilter/fault/fault_test.go

```diff
@@ -132,9 +122,8 @@ func clientSetup(t *testing.T) (*e2e.ManagementServer, string, uint32, func()) {
 		}
 	}()
 
-	return fs, nodeID, uint32(lis.Addr().(*net.TCPAddr).Port), func() {
-		fs.Stop()
-		bootstrapCleanup()
+	return managementServer, nodeID, uint32(lis.Addr().(*net.TCPAddr).Port), func() {
+		managementServer.Stop()
```

Comment:
We dont need this anymore since StartManagementServer already adds that to t.Cleanup()?

Reply:
Yes, removed.